bot.js

```diff
@@ -52,6 +58,72 @@ function postSvgTweet(status, svgString, altText) {
 		.catch(e => logError('Failed to post tweet: ' + e));
 }
 
+function resolveCssUrls(html) {
```

Comment:
Anything not directly related to the bot should be moved to another file (probably a new one)